script/constants/hostnames.js

```diff
@@ -0,0 +1,10 @@
+const environments = require('./environments');
+
+module.exports = {
+  [environments.DEVELOPMENT]: 'https://dev.vets.gov',
+  [environments.PREVIEW]: 'https://preview.va.gov',
+  [environments.PRODUCTION]: 'https://www.vets.gov',
+  [environments.STAGING]: 'https://staging.vets.gov',
+  [environments.VAGOVDEV]: 'https://dev.va.gov',
+  [environments.VAGOVSTAGING]: 'https://staging.va.gov'
```

Comment:
Should existing references to hostname [magic strings](https://github.com/department-of-veterans-affairs/vets-website/blob/7fb24d3df71cef7a029d62b9abda0946e73f1915/src/applications/personalization/profile360/vet360/util/local-vet360.js#L6-L7)/[constants](https://github.com/department-of-veterans-affairs/vets-website/blob/2e282e7d54c6b3ee70fb2c9d16138250bc9f2346/src/platform/brand-consolidation/index.js#L3-L4) now make use of these instead?

Reply:
I asked @ncksllvn about this yesterday, I'm not sure if we have a good place to put constants that can be shared between the build scripts and the application code.